Plugins/Statistics/GitStatistics/FormGitStatistics.cs

```diff
@@ -190,7 +190,7 @@ public void LoadLinesOfCode()
             void LoadLinesOfCodeForModule(IGitModule module)
             {
                 var filesToCheck = module
-                    .GetTree(null, full: true)
+                    .GetTree(module.RevParse("HEAD"), full: true)
```

Comment:
If `GetTree` never returns anything for `null`, perhaps it should throw.

Or, `null` means `HEAD` and you move the `RevParse` call inside `GetTree`. I'm guessing this is what used to happen before the regression.

Reply:
I think it should just not return a result.  I did it this way for a few reasons:

- It's clear what ObjectID we are sending in.
- Plugins SHOULD be able to call revparse.  

I disagree and did it with readability in mind and adding the revparse as a feature for any use of IGitModule.

Reply:
```bash
git ls-tree -z -r
git ls-tree -z
```
both return usage.  The commit is required.

null objectid should be reserved for invalid and not represent a valid objectid.

Reply:
No.  Before it was "head"  It was changed it to null when it was switched to objectid but null was never figured out to what it was.  I noticed I couldn't use module.revparse from the interface and thought that was a a missing feature.

Reply:
Supporting, both as to avoid null as implicit and to extend rev-parse for plugins

Reply:
Because.  It is not an exception.  It is an empty result.  If the parse of head say on a fresh repo with no commits it would be a valid empty set.  

![image](https://user-images.githubusercontent.com/3629489/47444964-b8738200-d785-11e8-8536-c85240e001cc.png)

Reply:
WELL that's a bug.  The commit tab should be empty after creating a new repo and opening that repo.

Reply:
Of course that is beta 1.  Not sure if master has the same issue.

Reply:
I think that it is a bug. Please file an issue for it.

Reply:
Sure. I am as going to check master to see if it is still a bug. I'll file one if it is.